packages/calling-component-bindings/src/handlers/createCommonHandlers.ts

```diff
@@ -800,7 +800,7 @@ export const createDefaultCommonCallingHandlers = memoizeOne(
       }
 
       if (togetherModeStreams.mainVideoStream.view) {
-        callClient.disposeView(call.id, togetherModeStreams.mainVideoStream);
+        callClient.disposeView(call.id, undefined, togetherModeStreams.mainVideoStream);
```

Comment:
Good catch on this, wonder how we missed this previously
![image](https://github.com/user-attachments/assets/63094381-2ff3-4b4a-8644-96abee29b59d)